schemas/context/person-name.schema.json

```diff
@@ -36,7 +36,7 @@
           "description":
             "Normally an abbreviation of a persons *title*, *honorific*, or *salutation*.\nThe `courtesyTitle` is used in front of full or last name in opening texts.\ne.g Mr. Miss. or Dr J. Smith.\n"
         },
-        "xdm:name": {
+        "xdm:fullName": {
           "title": "Full name",
           "type": "string",
           "description":
```

Comment:
We need to improve the descriptions.
I propose:
"The preferred last name of the person."

Reply:
@cdegroot-adobe I think you meant to make this comment on `xdm:lastName`, not `xdm:fullName`.